metplus/wrappers/ascii2nc_wrapper.py

```diff
@@ -48,6 +48,8 @@ def create_c_dict(self):
         c_dict['OBS_INPUT_DIR'] = self.config.getdir('ASCII2NC_INPUT_DIR', '')
         c_dict['OBS_INPUT_TEMPLATE'] = self.config.getraw('filename_templates',
                                                           'ASCII2NC_INPUT_TEMPLATE')
+        if not c_dict['OBS_INPUT_TEMPLATE']:
+            self.log_error("OBS_INPUT_TEMPLATE required to run")
```

Comment:
You should use the configuration variable name, not the c_dict name.
```suggestion
            self.log_error("ASCII2NC_INPUT_TEMPLATE required to run")
```